src/components/AutoAdvanceToggle/AutoAdvanceToggle.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useManifestState, useManifestDispatch } from '../../context/manifest-context';
+import './AutoAdvanceToggle.scss';
+
+const AutoAdvanceToggle = ({ label = "Autoplay", showLabel = true }) => {
+  const { manifest, autoAdvance } = useManifestState();
```

Comment:
Here, `manifest` can be removed since we don't use it in this component anymore?

Reply:
Good catch!